src/components/UpdateTokenModal.tsx

```
@@ -103,25 +105,21 @@ export function UpdateTokenModal({
   return (
     <div className="modal-overlay" onClick={handleClose}>
       <div className="modal-content" onClick={(e) => e.stopPropagation()}>
-        <h2>更新 Token</h2>
+        <h2>{t("update_token.title")}</h2>
 
-        <p className="modal-desc">
-          为账号 <strong>{accountName}</strong> 更新 Token。
-          <br />
-          <small>请确保新 Token 属于同一个用户，否则更新会失败。</small>
-        </p>
+        <p className="modal-desc" dangerouslySetInnerHTML={{ __html: t("update_token.desc", { name: accountName }) + "<br /><small>" + t("update_token.desc_note") + "</small>" }} />
```

Biome (2.4.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

(lint/security/noDangerouslySetInnerHtml)